apollo_fpga/commands/cli.py

```diff
@@ -16,12 +16,21 @@
 import logging
 import argparse
 from collections import namedtuple
+import xdg.BaseDirectory
+from functools import partial
 
 from apollo_fpga import ApolloDebugger
 from apollo_fpga.jtag import JTAGChain, JTAGPatternError
-from apollo_fpga.ecp5 import ECP5_JTAGProgrammer
+from apollo_fpga.ecp5 import ECP5_JTAGProgrammer, ECP5FlashBridgeProgrammer
 from apollo_fpga.onboard_jtag import *
 
+from amaranth.build.run import LocalBuildProducts
+try:
+    from luna.gateware.platform import get_appropriate_platform
+    from apollo_fpga.gateware.flash_bridge import FlashBridge, FlashBridgeConnection
+except ImportError:
+    pass
```

Comment:
This will stop the import failing, but if the user runs `apollo flash-fast` they'll just get a `NameError` later, right?

We should set a flag here, and if they try to use the feature, spot that flag and print a helpful message.